site/content/en/docs/Guides/feature-stages.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Feature Stages"
+linkTitle: "Feature Stages"
+date: 2019-09-26T01:20:41Z
+weight: 5
+description: >
+  This page provides a description of the various stages that Agones features can be in, and the
+  relative maturity and support level expected for each level.
+---
+
+A feature within Agones can be in `Alpha`, `Beta` or `Stable` stage.
+
+## Feature Gates
+
+`Alpha` and `Beta` features can be enabled or disabled through the `agones.featureGate` configuration option 
+that can be found in the [Helm configuration]({{< ref "/docs/Installation/helm.md#configuration" >}}) documentation.
+
+The current set of `alpha` and `beta` feature gates are:
+
+| Feature | Default | Stage | Since |
+|---------|---------|-------|-------|
+| None currently | 
```

Comment:
Please add multicluster allocation to this table. Even if we put an asterisk and say it was grandfathered in before the process was put into place, it's clearly alpha stage (by the api definition) and we should call that out here.

Reply:
Done!